File: DivinoLanches/ClientApp/src/models/ProdutoModel.ts
export class ProdutoModel {

    id: Number
    nome: string
    tipo: string
    subTipo: string
    valor: string
    ativo: string

    constructor(obj: any) {
        this.id = obj.id
        this.nome = obj.nome
        this.tipo = obj.tipo
        this.subTipo = obj.subTipo
        this.valor = obj.valor       
        this.ativo = obj.ativo       
        return this
    }
}

File: DivinoLanches/Models/ProdutoModel.cs
using System;

namespace DivinoLanches.Models
{
    public class ProdutoModel
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Tipo { get; set; }
        public string SubTipo { get; set; }
        public string Valor { get; set; }
        public bool? Ativo { get; set; }
    }
}
